### ISSUE TITLE
Add credit for Diversity, Equity, Inclusion image for redesigned homepage

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3353]
### Overview
We need to add a credit for a new image used in the redesigned version of the homepage so that the Credits webpage is up to date.

### Details
From the Homepage Redesign Credits in Figma, here is a screenshot of the credit's information for reference:
    <details>
      <summary>Click here to see the screenshot of the Diversity, Equity, Inclusion credit in Figma</summary>
      <img alt="" src="https://user-images.githubusercontent.com/31293603/178171557-b83a7773-952f-466d-8af5-e0ba719bd745.png">
    </details>

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch`. Please follow ["Step 1: Create your branch" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-1-create-your-branch) wiki article.
- [x] In `_data/internal/credits` directory, create a file called `diversity-equity-inclusion.yml`
- [x] In the file you just created, copy and paste the following:
```
---
# The title from the source provider
title: Diversity, Equity, Inclusion
# The url where you can get direct access to the image on the provider site
title-link: https://burst.shopify.com/photos/team-hands-middle?q=matthew+henry+diversity
# The type of content - image, audio, or video 
content-type: image
# The page it's used on
used-in: Homepage
# Name of the artist
artist: Matthew Henry
# Name of the image database you found it on
provider: Burst
# Please give the general site URL, not the link to the image
provider-link: https://burst.shopify.com/
# Location in our GitHub repo
image-url: /assets/images/program-areas/diversity-equity-inclusion.png
# Alt text for the image on the Credits page
alt: ''
# If an image was altered, please give the name of the HfLA designer that altered the image
altered-by: Kristine Eudey
---
```
- [x] To update your topic branch, please follow ["Step 2: Update your branch with upstream changes" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-2-update-your-branch-with-upstream-changes) wiki article.
- [x] The pull request created from this issue should be merged into `feature-homepage-launch`. Please follow ["Step 3: Create a pull request" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-3-create-a-pull-request) wiki article
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3130 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please do the following in issue #3130:
    - [x] Move it to the New Issue Approval column
    - [x] Remove the "Dependency" label from it
    - [x] Add a "ready for dev lead" label to it  
  
### Resources/Instructions
- [Credits webpage](https://www.hackforla.org/credits/)
- [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724)
- [How to work off of a feature branch wiki page](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [Credits yml file template for Iconography and Imagery](https://github.com/hackforla/website/wiki/Credits-yml-file-template-for-Iconography-and-Imagery)
- This issue is tracked in the epic #3130.

**Comment by JessicaLucindaCheng** [comment]
This issue would usually have the milestone 01. Compliance. However, it's specifically for the homepage launch, which has not happened yet.

**Comment by JessicaLucindaCheng** [comment]
There are some differences between an actual image we use and the one we link to on the Credits page. I think the image may have been edited by a HfLA designer by looking at previous version of the redesigned homepage in Figma: https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15962%3A100862.
1. How do we tell if it is edited by a HfLA designer?
2. If it it is edited by a HfLA designer, how do we go about giving credit to an edited image that is a combination of images?
3. If it it is edited by a HfLA designer, do we need to give credit to each individual icon/image within the overall image too?

For this credit, the link for the image is: https://burst.shopify.com/photos/team-hands-middle?q=matthew+henry+diversity
  <details>
    <summary>Click here to see the image from Burst</summary>
    <img src="https://user-images.githubusercontent.com/31293603/180334642-3cf45c33-9e40-4f40-8b2d-0496570965e0.jpg" alt="team hands in the middle">
  </details>

It doesn't look exactly like the contents of this image we are using: 
  <details>
    <summary>Click here to see the screenshot of the marked up Diversity, Equity, Inclusion credit in Figma</summary>
    <img width="553" alt="" src="https://user-images.githubusercontent.com/31293603/180334762-86967c48-3ff9-4f31-83b4-f8ad5383696a.png">
  </details>

Ans from Bonnie: Add an altered-by field and give credit to the HfLA designer that altered the image.

A separate issue will handle displaying the logic for this to show up on the Credits webpage.

**Comment by JessicaLucindaCheng** [comment]
Note: The "Altered by" field will not show up for the credit when you view the Credits page using Docker. The code to have the "Altered by" field viewable will be implemented in a separate issue.

**Comment by str-xjua24** [comment]
Availability: 12/7-12/10 Afternoons
ETA: End of the week
